app/views/result_filters/location/new.html.erb

```diff
@@ -14,31 +14,76 @@
         </legend>
 
         <div class="govuk-form-group">
-          <% if flash[:error] %>
-            <span id="location-error" class="govuk-error-message">
-              <span class="govuk-visually-hidden">Error:</span> Please choose an option
-            </span>
-          <% end %>
-          <div class="govuk-radios">
+          <div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios">
             <div class="govuk-radios__item">
               <%=
                 form.radio_button(
                   :l,
-                  "2",
+                  "1",
                   class: "govuk-radios__input",
-                  data: { qa: "across_england" },
-                  checked: params[:l] == "2"
+                  data: { qa: "by_postcode_town_or_city" },
+                  checked: params[:l] == "1",
+                  aria: { 'controls': "location-conditional" }
                 )
               %>
-              <%= form.label :l_2, "Across England", class: "govuk-label govuk-radios__label" %>
+              <%= form.label :l_1, "By postcode, town or city", class: "govuk-label govuk-radios__label" %>
+            </div>
+            <div class="govuk-radios__conditional <%= "govuk-radios__conditional--hidden" unless flash[:location_error] %>" id="location-conditional">
+              <div class="govuk-form-group">
+                <%= form.label :lq, "Postcode, town or city", class: "govuk-label" %>
+                <div class="govuk-error-message" data-qa="location-error"><%= flash[:location_error] %></div>
```

Comment:
`data-qa="location-error"`

that needs to be id

as clicking on the link

![image](https://user-images.githubusercontent.com/470137/73643422-64836880-466b-11ea-9d24-d6e76f1d321a.png)

do not jump to section

Reply:
Cool 👍 
Have pushed fix